### PR TITLE
Update compat_rhs_usf3 (Weapons/Ammo)

### PR DESCRIPTION
**Pull request by J3aLeiNe**
Update CfgWeapons.hpp : 
 - add M24
 - add M107
 - add HK416 10" and 14.5"
 - add M249S and L
 - add M27 IAR
 - add MP7
 - add Glock17
 - add M9
 - update with the new overheating format :
https://github.com/acemod/ACE3/blob/master/addons/overheating/CfgWeapons.hpp
 - add Premier Reticles scopes

Update CfgAmmo.hpp :
 - add M107 ammos
 - add  5,56x45 M855A1
 - add 4,6x30 FMJ, AP and JHP 
 - add 9mm FMJ and JHP

### CODE REVIEW

**Comment by J3aLeiNe** [comment]
I'm not a RHS user, not tested sorry.

File: optionals/compat_rhs_usf3/CfgAmmo.hpp [hunk 1]
```diff
@@ -2,6 +2,31 @@
 class CfgAmmo
 {
     class BulletBase;
+    class B_127x99_Ball;
+    class rhsusf_ammo_127x99_M33_Ball: B_127x99_Ball {
+        ACE_caliber=12.954;
```
Comment (jonpas) [inline optionals/compat_rhs_usf3/CfgAmmo.hpp]: Spaces before and after `=` almost everywhere.

File: optionals/compat_rhs_usf3/CfgAmmo.hpp [hunk 2]
```diff
@@ -2,6 +2,31 @@
...
+        ACE_bulletLength=58.674;
+        ACE_bulletMass=42.83;   //661gr - Barret technical data
```
Comment (jonpas) [inline optionals/compat_rhs_usf3/CfgAmmo.hpp]: One space between entry and comment.

**Comment by J3aLeiNe** [comment]
Spaces Wars today, sorry for my mistakes. :)

**Comment by J3aLeiNe** [comment]
Need to be tested by a RHS user.

**Comment by PabstMirror** [comment]
Will fix #4162

**Comment by J3aLeiNe** [comment]
Question about the 12.7x99 ammos, the best at your opinion : the ACE3 default M33 included in the AtragMx or a custom-made M33 a little bit more accurate but not included in the AtragMx ?
What is better for the player and the gameplay ?

**Comment by esteldunedain** [comment]
IMO the default M33

**Pull request merged**